docs/parts/actuators.md

```diff
@@ -194,6 +194,30 @@ Some very inexpensive toy cars use a DC motor to drive the back wheels forward a
 ```
 A PCA9685 could also be used to generate all control signals.  See [pins](pins.md) for a detailed discussion of pin providers and pin specifiers.
 
+## VESC for both Steering and Throttle
+
+VESC is an advanced version of ESC that provides you a lot of customization options on how the ESC operates. It includes features such as regenerative braking, temperature control etc.
+
+This was tested with a VESC 6 and Traxxas Brushless Motor
+Follow this tutorial to update your VESC firmware and calibrate it: https://f1tenth.readthedocs.io/en/stable/getting_started/firmware/firmware_vesc.html
```

Comment:
I've read the linked instructions.  They include using this [motor configuration XML](https://drive.google.com/file/d/1-KiAh3hCROPZAPeOJtXWvfxKY35lhhTO/view).  Is this very specific to the F1Tenth build or is it generic enough for any BLDC?  What if I have some motor I bought (which I do); is the configuration suitable or do I need to modify it or create a new one so I don't blow up my motor?

Reply:
Will check this out. Thanks for bringing it up!